src/xpk/core/core.py

```diff
@@ -2104,8 +2104,12 @@ def get_main_container(args, system, docker_image, resource_type) -> str:
         ' is required but not installed. Aborting"; exit 24;};'
```

Comment:
I don't think you need the gsutil_test_command anymore since you move to gcloud storage

Reply:
Good point. Let me change this part accordingly.